sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -619,670 +490,11 @@ def format(self):
     return 'bigquery'
 
   def writer(self, test_bigquery_client=None, buffer_size=None):
-    return BigQueryWriter(
+    return bigquery_tools.BigQueryWriter(
         sink=self, test_bigquery_client=test_bigquery_client,
         buffer_size=buffer_size)
 
 
-# -----------------------------------------------------------------------------
-# BigQueryReader, BigQueryWriter.
-
-
-class BigQueryReader(dataflow_io.NativeSourceReader):
```

Comment:
I suggest adding stubs (with warnings) for BigQueryReader (and other public classes deleted here) as well. We can delete them after few releases.

Reply:
Done! thanks.